src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,81 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.exception.RegistrationException;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static final int incorrectAge = 15;
+    private static final int DefaultAge = 18;
+    private static final String incorrectPassword = "afdsf";
+    private static final String DefaultPassword = "abca4567";
+    private static final String DefaultLogin = "username@gmail.com";
+    private static final String incorrectLogin = "abas";
+    private static RegistrationServiceImpl registrationService;
+    private static final User user = new User();
+
+    @BeforeAll
+    static void beforeAll() {
+        registrationService = new RegistrationServiceImpl();
+    }
+
+    @BeforeEach
+    void beforeEach() {
+        user.setAge(DefaultAge);
+        user.setLogin(DefaultLogin);
+        user.setPassword(DefaultPassword);
+    }
+
+    @Test
+    void checkisUser_Null() {
```

Comment:
1. What 'is' means in most of cases naming? Please remove
2. Please use some better pattern to show method called, input details and expected result
like
registerUser_nullPassword_NotOk()
or something what you like but more informative than it is now

Reply:
Renamed